ethereum/executionlayer/build.gradle

```diff
@@ -10,6 +10,9 @@ dependencies {
 
   implementation 'com.google.code.gson:gson'
   implementation 'org.apache.tuweni:tuweni-units'
+  implementation 'io.jsonwebtoken:jjwt-api'
+  implementation 'io.jsonwebtoken:jjwt-impl'
+  implementation 'io.jsonwebtoken:jjwt-jackson'
```

Comment:
I think these two should be `runtimeOnly` so they aren't on the compile class path and we're forced to only use the stable API.